features/netsocket/cellular/generic_modem_driver/PPPCellularInterface.cpp

```diff
@@ -262,6 +262,7 @@ PPPCellularInterface::PPPCellularInterface(FileHandle *fh, bool debug)
     _pwd = NULL;
     _fh = fh;
     _debug_trace_on = debug;
+    _stack = IPV4_STACK;
```

Comment:
I set the value to default stack. In that case flag configuration defines the stack.